server/chart_config.json

```diff
@@ -57,6 +57,22 @@
     "isOverview": true,
     "isChoropleth": true
   },
+  {
+    "category": "Economics",
+    "titleId": "Worker vs non-worker population",
+    "title": "Worker vs non-worker population",
```

Comment:
non-worker is one word https://www.thefreedictionary.com/nonworker

"Worker vs. Nonworker Population"

?

Reply:
Title-case titles other places too?

Reply:
The place page has not been using title case. I will not change it for now. If needed, we can do an overall conversion.